clear/main.py

```diff
@@ -194,6 +194,12 @@ def pfccounters():
     command = "pfcstat -c"
     run_command(command)
 
+@cli.command()
+def dropcounter():
```

Comment:
Suggest pluralizing the subcommand name, "dropcounters". The Click library will handle abbreviation and tab-completion, so "dropcounter" will still work. This suggestion would apply to all commands in this PR.